index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Twitter</title>
+</head>
+<body>
+  <header class="header">
```

Comment:
Good idea using sections, headers, h1s, h2s, etc. Classes are also named really nice and organized, comments in the css help a lot as well. Makes your code and structure easy to read.